girder/utility/server.py

```diff
@@ -18,16 +18,34 @@
 ###############################################################################
 
 import cherrypy
+import mako
 import mimetypes
 import os
 import six
 
 import girder.events
 from girder import constants, logprint
+from girder.api.describe import API_VERSION
 from girder.utility import plugin_utilities, model_importer
 from girder.utility import config
 from . import webroot
 
+# TODO make a pretty error page someday.
+_errorTemplate = """
```

Comment:
I'd rather see this broken out to a static mako template for consistency with our other makos.  I'd be ok with a TODO comment or a GH issue tracking that though.